app/assets/stylesheets/exercises/show.css.scss

```diff
@@ -62,6 +62,59 @@
   position: fixed;
   left: calc(33%);
 
+  #success, #fail {
+    display: none;
+    position: absolute;
+    top:0;
+    height: 4.6em;
+    width: 100%;
+    border: 0.2em solid $white;
+    padding-left: 1.5em;
+    padding-top:1em;
+    color: $white;
+  }
+
+  #success {
+    background-color: $turquoise;
+    #next, p {
+      display: inline-block;
+    }
+
+    #next {
+      margin-left: 3em;
+      a {
+        background-color: $red;
+        padding: .65em 1em;
+        color: $white;
+      }
+    }
+
+    p {
+      font-size: 1.5em;
+    }
+  }
+
+  #fail {
+    background-color: $red;
+    padding-top: 1.3em;
+  }
+
+  #close {
+    cursor: default;
+    position:absolute;
+    right: 1em;
+    top: 1em;
+    color: $white;
+    line-height: 0em;
+     &:hover {
+        color:black;
+    }
+  }
+
+  #message, #key_words {
+    display: none;
```

Comment:
I usually have one class with the styling `display: none` and apply that class to the elements that need them